src/PowerShellEditorServices/Session/Host/EditorServicesPSHostUserInterface.cs

```diff
@@ -684,6 +688,60 @@ private void WriteDebuggerBanner(DebuggerStopEventArgs eventArgs)
             }
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        public static ConsoleColor BackgroundColor
```

Comment:
All of the color properties here can be `internal`, the empty documentation comments can then be removed as well.